app/controllers/reporting_controller.rb

```diff
@@ -4,6 +4,8 @@ def current_salary_by_graduation_year
   end
 
   def salaries_by_years_of_experience
-    render json: SalaryReportingService.salaries_by_years_of_experience
+    records = SalaryReportingService.salaries_by_years_of_experience
```

Comment:
I re-used this SQL query, and didn't include caching to avoid scope creep. We could look at optimizing a specialized query for chart data in the future.